representation/logical-probe.metta

```diff
@@ -0,0 +1,24 @@
+;;;;;;;;;; Logical Probe ;;;;;;;;;;;
+;; Builds a tuple of LSKs by processing the sample logical perms 
+;; Params:
+;;   $exemplar: Reference tree containing the target node.
+;;   (mkNodeId $targetId): ID of the target node in the exemplar.
+;;   $perms: Tuple of expressions returns from the sampleLogicalPerms.
+;;   $addIfInExemplar: If true, include knobs even if in exemplar.
+
+;; Return: LSKs
+
+(: logicalProbe (-> (Tree $a) NodeId Expression Bool Expression))
+(= (logicalProbe $exemplar (mkNodeId $targetId) () $addIfInExemplar) ())
```

Comment:
I think $this line is redundent. The logic below will also handle the empty case of the `$perms` correctly.